internal/component/faro/receiver/internal/payload/payload.go

```diff
@@ -238,6 +238,14 @@ func (m Measurement) KeyVal() *KeyVal {
 	}
 	MergeKeyVal(kv, m.Trace.KeyVal())
 	MergeKeyValWithPrefix(kv, KeyValFromMap(m.Context), "context_")
+
+	values := make(map[string]float64, len(m.Values))
+	for key, value := range m.Values {
+		values[key] = value
+	}
```

Comment:
Why are we copying this? Can we not just call `KeyValFromFloatMap(m.Values)`?

Reply:
true, it's an artifact from a previous implementation that had a conversion there. Let me spawn a quick pr to remove this